src/audio.c

```diff
@@ -1906,6 +1937,12 @@ void audio_sdp_attr_decode(struct audio *a)
 			}
 		}
 	}
+	/*
+	EBUACIP handler
+	EBU TECH 3368 profile provisioning on incomming invite.
+	*/
+	sdp_media_rattr_apply(stream_sdpmedia(a->strm),
+	 "ebuacip", ebuacip_handler, a);
```

Comment:
you might want to consider if this should be protected by the ebuacip config option.

```
sdp_ebuacip    yes|no
```